plugins/python/src/main/kotlin/io/paddle/plugin/python/utils/PipArgs.kt

```diff
@@ -21,6 +22,10 @@ internal class PipArgs private constructor(val args: List<String>) {
             if (noCacheDir) {
                 add("--no-cache-dir")
             }
+            if (noBinaryList.isNotEmpty()) {
+                add("--no-binary")
+                add(noBinaryList.joinToString(separator = ","))
+            }
             addAll(additionalArgs)
             addAll(packages)
```

Comment:
shouldn't we drop the packages which are "noBinary" from this list here? otherwise, we would mention them 2 times?

Reply:
Yes, this is some "format option" to that option, see: [link](https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-no-binary) and example below